terragraph_planner/common/rf/link_budget_calculator.py

```diff
@@ -185,7 +185,7 @@ def extract_gain_from_radio_pattern(
         DataException,
     )
 
-    # Convert deviation from (-180, 180] to [0, 360)
```

Comment:
Are you sure about this one?

Reply:
If 180 is exclusive in before adding 180, 360 should be inclusive too after adding, right?

Reply:
180 is included. 360 is not. [0,360) means 0 included, 360 excluded.

Reply:
I know, but I think it's wrong because:
- if `ez_deviation` is 180, it's valid because 180 is included, but after line 189-193, it becomes 360, which is invalid because 360 is excluded. That's not inconsistent.

Reply:
180 is >= 0 so it doesn't become 360. Only negative values get 360 added to it.

Reply:
My bad. You're correct. I will discard this PR, and let you fix the other minor typo in your PR #25 . Sounds good?